### PR TITLE
Fix transit-vehicle-overlay deps/coreUtils imports

### PR DESCRIPTION
**Pull request by landonreed**
This PR:
- removes some seemingly unneeded deps
- fixes some coreutils imports to adhere to import from root export as suggested by https://github.com/opentripplanner/otp-ui/pull/256

### CODE REVIEW

File: packages/transit-vehicle-overlay/src/components/popups/CustomTooltip/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import PropTypes from "prop-types";
 import { Tooltip } from "react-leaflet";
 import L from "leaflet";
 
-import { transitVehicleType } from "@opentripplanner/core-utils/src/types";
+import coreUtils from "@opentripplanner/core-utils";
```
Comment (binh-dam-ibigroup) [inline packages/transit-vehicle-overlay/src/components/popups/CustomTooltip/index.js:6]: Move this import to the top of the file.

File: packages/transit-vehicle-overlay/src/components/popups/VehiclePopup/index.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Popup } from "react-leaflet";
+import coreUtils from "@opentripplanner/core-utils";
```
Comment (binh-dam-ibigroup) [inline packages/transit-vehicle-overlay/src/components/popups/VehiclePopup/index.js:4]: Move this import to the top of the file.

File: packages/transit-vehicle-overlay/src/components/popups/vehicle-tracker.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { transitVehicleType } from "@opentripplanner/core-utils/src/types";
+import coreUtils from "@opentripplanner/core-utils";
```
Comment (binh-dam-ibigroup) [inline packages/transit-vehicle-overlay/src/components/popups/vehicle-tracker.js:3]: Move import to the top of the file.

File: packages/transit-vehicle-overlay/package.json [hunk 4]
```diff
@@ -9,13 +9,10 @@
   "module": "esm/index.js",
   "private": false,
   "dependencies": {
-    "@opentripplanner/base-map": "^1.0.4",
```
Comment (binh-dam-ibigroup) [inline packages/transit-vehicle-overlay/package.json:12]: This is interesting, Storybook seems to display the map fine for this package's stories, even without this import.

**Review by binh-dam-ibigroup — APPROVED** [review]
Thanks @landonreed for tracking down the faulty imports! Please fix the import order then this PR should be good to go.

**Review by evansiroky — APPROVED** [review]
Approved pending of merging in of https://github.com/ibi-group/otp-ui/pull/6.

**Pull request merged**